docs/server/java/_reference.mdx

```diff
@@ -51,7 +51,9 @@ class StatsigOptions(
     var localMode: Boolean = false,
     var rulesetsSyncIntervalMs: Long = CONFIG_SYNC_INTERVAL_MS,
     var idListsSyncIntervalMs: Long = ID_LISTS_SYNC_INTERVAL_MS,
-    var dataStore: IDataStore? = null
+    var dataStore: IDataStore? = null,
```

Comment:
do we have the data store type as well?  might need to add that

Reply:
localDataFile havent release yet, so this is fine now. We can update next week